webui-src/app/theme.css

```diff
@@ -12,7 +12,7 @@ General site-wide rules
 #tab-content {
   height: 100%;
   /* Margin on top to avoid getting covered by navbar */
-  margin-top: 4em;
+  padding-top: 4em;
```

Comment:
I think it's better to be more consistent with the sidebar approach, the main nav bar and the nested sidebars should be on the left.